inference/core/version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.12.1"
+__version__ = "0.13.0rc2"
```

Comment:
will be 0.13 for release?

Reply:
yeah, will change that once merged into `development` branch or just before release